lib/msh3.cpp

```diff
@@ -6,9 +6,10 @@
 --*/
 
 #include "msh3.hpp"
+#include <atomic>
 
-const MsQuicApi* MsQuic;
-long MsH3RefCount = 0;
+static const MsQuicApi* MsQuic;
```

Comment:
```suggestion
const MsQuicApi* MsQuic;
```
This broke the build.

Reply:
I saw, just fixed it.